src/v0/timetracking.ts

```diff
@@ -173,6 +198,59 @@ export class Timetracking extends ThBaseHandler {
     }
   }
 
+  async getAllConfigurations (): Promise<TimetrackingConfigurationResponse> {
```

Comment:
The naming is misleading. It suggest that the response will be an array, but you're returning an object as the data.
In this case I would change the name of the method to `getConfiguration`